### PR TITLE
docs: update ps_accounts docs according installer one

### PR DESCRIPTION
**Pull request by emmanuelgautier**

### CODE REVIEW

File: docs/src/12-legacy-account-and-billing/README.md [hunk 1]
```diff
@@ -177,23 +177,25 @@ class Rbm_example extends Module {
...
+Replace the `<module_name>` by your module name in order to avoid conflicts.
+
 ```yaml
   # ...
 
   #####################
   # PS Account
 
-  ps_accounts.installer:
+  <module_name>.installer:
```
Comment (hschoenenberger) [inline docs/src/12-legacy-account-and-billing/README.md:188]:
```suggestion
  <module_name>.ps_accounts_installer:
```

File: docs/src/12-legacy-account-and-billing/README.md [hunk 2]
```diff
@@ -177,23 +177,25 @@ class Rbm_example extends Module {
...
     class: 'PrestaShop\PsAccountsInstaller\Installer\Installer'
     public: true
     arguments:
       - "5.0"
 
-  ps_accounts.facade:
+  <module_name>.facade:
```
Comment (hschoenenberger) [inline docs/src/12-legacy-account-and-billing/README.md:194]:
```suggestion
  <module_name>.ps_accounts_facade:
```

File: docs/src/12-legacy-account-and-billing/README.md [hunk 3]
```diff
@@ -177,23 +177,25 @@ class Rbm_example extends Module {
...
     class: 'PrestaShop\PsAccountsInstaller\Installer\Facade\PsAccounts'
     public: true
     arguments:
-      - "@ps_accounts.installer"
+      - "@<module_name>.installer"
```
Comment (hschoenenberger) [inline docs/src/12-legacy-account-and-billing/README.md:198]:
```suggestion
      - "@<module_name>.ps_accounts_installer"
```

File: docs/src/4-prestashop-account/README.md [hunk 4]
```diff
@@ -36,15 +36,15 @@ Integrating your module will require using [PHP](https://www.php.net/) for the b
 
       #####################
       # PrestaShop Account
-      ps_accounts.installer:
+      <module_name>.installer:
```
Comment (hschoenenberger) [inline docs/src/4-prestashop-account/README.md:39]: Same remark as above and also applies below

**Review by hschoenenberger — CHANGES_REQUESTED** [review]

File: docs/src/4-prestashop-account/README.md [hunk 5]
```diff
@@ -36,15 +36,15 @@ Integrating your module will require using [PHP](https://www.php.net/) for the b
...
         class: 'PrestaShop\PsAccountsInstaller\Installer\Installer'
         arguments:
           - '5.0'
 
-      ps_accounts.facade:
+      <module_name>.ps_accounts_facade:
         class: 'PrestaShop\PsAccountsInstaller\Installer\Facade\PsAccounts'
         arguments:
-          - '@ps_accounts.installer'
+          - '@<module_name>.installer'
```
Comment (hschoenenberger) [inline docs/src/4-prestashop-account/README.md:47]:
```suggestion
          - '@<module_name>.ps_accounts_installer'
```

File: docs/src/5-prestashop-billing/README.md [hunk 6]
```diff
@@ -37,15 +37,15 @@ Our team is here to help you get started with the implementation of PrestaShop B
...
         class: 'PrestaShop\PsAccountsInstaller\Installer\Installer'
         arguments:
           - '5.0'
 
-      ps_accounts.facade:
+      <module_name>.ps_accounts_facade:
         class: 'PrestaShop\PsAccountsInstaller\Installer\Facade\PsAccounts'
         arguments:
-          - '@ps_accounts.installer'
+          - '@<module_name>.installer'
```
Comment (hschoenenberger) [inline docs/src/5-prestashop-billing/README.md:48]:
```suggestion
          - '@<module_name>.ps_accounts_installer'
```

**Review by hschoenenberger — CHANGES_REQUESTED** [review]

File: docs/src/4-prestashop-account/README.md (hunk 5 above)
Comment (emmanuelgautier) [reply docs/src/4-prestashop-account/README.md]: My bad. Sorry

**Review by hschoenenberger — APPROVED** [review]

**Pull request merged**